cms/locale/cy/LC_MESSAGES/django.po

```diff
@@ -19,28 +19,29 @@ msgstr ""
 "Plural-Forms: nplurals=4; plural=(n==1) ? 0 : (n==2) ? 1 : (n != 8 && n != "
 "11) ? 2 : 3;\n"
 
-#: cms/articles/models.py:164 cms/core/enums.py:8
+#: cms/articles/models.py:173 cms/core/enums.py:8
 #: cms/core/formatting_utils.py:42
 msgid "Article"
 msgstr "Erthygl"
 
-#: cms/articles/models.py:396
+#: cms/articles/models.py:405
 msgid "Cite this article"
 msgstr "Dyfynnwch yr erthygl hon"
 
-#: cms/articles/models.py:398
+#: cms/articles/models.py:407
 #: cms/jinja2/templates/components/contact_details/contact_details.html:6
 #: cms/methodology/models.py:188 cms/release_calendar/models.py:246
 msgid "Contact details"
 msgstr "Manylion cyswllt"
 
-#: cms/articles/models.py:435
-msgid "All data related to"
-msgstr ""
+#: cms/articles/models.py:445
```

Comment:
It looks like this was generated before the final changes to the `models.py` file. The lines don't match (445 here but 444 in model.py). If you run `make makemessages` again, it will fix it. Then just commit the new changes.